pkgs/servers/web-apps/plausible/default.nix

```diff
@@ -36,26 +37,11 @@ let
     ];
   };
 
-  yarnDeps = mkYarnModules {
```

Comment:
Just to let you know. I have a pending PR on plausible to update it to 1.4.0, it's ready but needs approval. If you don't mind, you'll need to wait a bit until it's merged then you'll be able to rebase.

Reply:
I am aware of that, thanks :)